app/controllers/little_shop_app.rb

```diff
@@ -112,8 +120,4 @@ class LittleShopApp < Sinatra::Base
     erb :"/items/show"
   end
 
-  get '/api/v1/items/:id' do |id|
```

Comment:
This needs to stay in.

Reply:
Thanks for catching that, we definitely need it.